### DISCUSSION TITLE
월드가 끝날때 어떻게 나가게 해야하나요?

### DISCUSSION THREAD

**Discussion by PH62mem** [naverz/zepeto-studio-kor#918] [5. Q&A]
나가게 하는 코드가 잇나요?
아니면 월드 오버시에 강제로 나가게하거나 아바타를 제어하지 못하게 할 수 있나요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 간단하게 구현하시려면 월드 오버 씬을 만들고 그 씬에는 제페토 캐릭터를 등장시키지 않는 방법이 있습니다.
기본적으로 방을 나가는 방법은 직접 홈 UI에서 나가기 버튼을 사용자가 조작해야 합니다.

**Reply by PH62mem** [reply]
해당 게임이 멀티인 경우에도 동일하게 적용될까요?
